src/index.html

```diff
@@ -6,22 +6,432 @@
+        <a
+          href="#"
+          class="header__link"
+        >
+          <img
+            src="https://github.com/sinner1993/layout_moyo-header/blob/master/src/images/logo.png?raw=true"
+            alt="moyo logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!--#endregion -->
+    <main class="main">
+      <!--#region card  -->
+      <div
+        data-qa="card"
+        class="card"
+      >
+        <div class="card__photo">
+          <img
+            src="./images/imac.jpeg"
+            alt="laptop"
+            class="card__img"
+          />
+        </div>
+        <div class="card__main">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+        </div>
+        <div class="card__rate">
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <h5 class="card__h5">Price:</h5>
+          <p class="card__sum">$2,199</p>
+        </div>
+        <div class="card__link">
```

Comment:
Redundant
```suggestion
```

Reply:
when I m removing  that then code  is broken

Reply:
Not fixed, you need to remove it, because this div is redundant
```suggestion
```